src/app/dashboard/dashboard.component.ts

```diff
@@ -37,15 +37,23 @@ export class DashboardComponent implements OnInit {
   ngOnInit() {
     this.connectionStore.loadAll();
     this.integrationStore.loadAll();
+
     /**
      * If guided tour state is set to be shown (i.e. true), then show it for this page, otherwise don't.
      */
     if (this.userService.getTourState() === true) {
       this.tourService.initialize([ {
-          anchorId: 'dashboard.navigation',
+        route: 'dashboard',
+        anchorId: 'dashboard.navigation',
+          title: 'Navigation',
           content: 'View integrations, connections or settings for applications that Fuse Ignite is registered with.',
           placement: 'right',
-          title: 'Navigation',
+        }, {
+        route: 'dashboard',
```

Comment:
same here with this line and the one after

Reply:
That's one of the things I wasn't sure about.